build_product_images.py

```diff
@@ -85,26 +99,55 @@ def build_and_publish_image(args, product):
     """
     Returns a list of commands that need to be run in order to build and
     publish product images.
+
+    For local building, builder instances are supported.
     """
     commands = []
-
+    push = ""
     image_name = f'{args.registry}/stackable/{product["name"]}'
     tags = build_image_tags(image_name, args.image_version, args.product_version)
     build_args = build_image_args(product["versions"][0])
 
-    commands.append(
-        [
-            "docker",
-            "build",
-            *build_args,
-            *tags,
-            "-f",
-            product["name"] + "/Dockerfile",
-            ".",
-        ]
-    )
     if args.push:
-        commands.append(["docker", "push", "--all-tags", image_name])
+        push = "--push"
+
+    # Multiarch builds
+    if args.multiarch:
+        commands.append(
+            [
+                "docker",
+                "buildx",
+                "build",
+                *build_args,
+                *tags,
+                "-f",
+                product["name"] + "/Dockerfile",
+                "--platform",
+                "linux/amd64,linux/arm64",
```

Comment:
Just use the contents of `args.architecture` here.

Reply:
Done